tests/dialects/test_func.py

```diff
@@ -71,8 +71,8 @@ def test_func_II():
     # Use this region to create a func0
     func1 = FuncOp.from_region("func1", [], [], region0)
 
-    ops0 = list(func1.regions[0].blocks[0].ops)
-    ops1 = list(func1.regions[0].blocks[1].ops)
+    ops0 = list(list(func1.regions[0].blocks)[0].ops)
+    ops1 = list(list(func1.regions[0].blocks)[1].ops)
```

Comment:
This seems to be a strict downgrade, can we phrase this differently?

Reply:
I thought that since this was just a test, I wouldn't make things too weird, but I'm happy to rewrite if you think it's too messy